tests/test_person_search.py

```diff
@@ -0,0 +1,55 @@
+# coding: utf8
+import time
+from model.user import User
+from pages.persons.persons_page import PersonsPage
+__author__ = 'vika'
+
+
+GIVEN_SURNAME = u'Іваненко'
+GIVEN_PERSON_ID = '14'
+GIVEN_NUM_OS = '34654654'
+GIVEN_SERIES_OS = '32'
+
+SLEEP_TIME = 7
+
+def test_is_login(app):
```

Comment:
We have the same test in other place (it is actually authorization test). Do we need it here?